### PR TITLE
Updating yaml pipeline and parallel security tasks

### PR DESCRIPTION
**Pull request by ramourt**
Parallelizing security and build tasks
Moving arrow SDL retention to release pipeline
Moving cloudroll deployment to release pipeline

### CODE REVIEW

**Comment by erinha** [comment]
<p>Analyzed commit id : 4a7e6d559ae01bd980e4e83c9ab01f959837ea4b</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.43 KB</td><td>59.42 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: 1ae1c726af07ed6f4c9f41264f7c48fe81a558d6</p>

**Comment by MarcinJuraszek** [comment]
Oh, I didn't know that's a thing. Would be great to do it in the other repo as well! :)

**Comment by erinha** [comment]
<p>Analyzed commit id : 460a62aa71cb8a9927aef77307514e6122dcd059</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.38 KB</td><td>59.37 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.38 KB</td><td>59.37 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: fc697ff3da1a191a40d7a947afb7f45b50c5e0a6</p>

**Comment by ramourt** [comment]
> Oh, I didn't know that's a thing. Would be great to do it in the other repo as well! :)

Yes, I was looking at it and we can definitely do that. 
I was also wondering if it might be a good idea to merge all the yaml files into one. The e2e tests can be added in here as a separate stage.

**Comment by erinha** [comment]
<p>Analyzed commit id : c83a3228ae88dd17f9d39b5fe074549f8a1e669c</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -7 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>59.38 KB</td><td>59.37 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr><tr><td>Total Size</td><td>59.38 KB</td><td>59.37 KB</td><td><span style="color: green">⯆</span> -7 Bytes</td></tr></table></details><hr><p>Baseline commit: 4e3f05ee3569953a280f1b7bba41332ef3db2200</p>

File: azure-pipelines.yml [hunk 1]
```diff
@@ -14,254 +14,237 @@ trigger:
...
-    condition: and(
-      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
-      eq(variables['Build.SourceBranch'], 'refs/heads/2.0-preview'))
-    inputs:
-      ConnectionType: 'AzureRM'
-      azureSubscription: '$(AzureSubscription)'
-      appType: 'webApp'
-      WebAppName: 'cloudroll'
-      package: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
-    displayName: 'Deploy to cloudroll'
```
Comment (erinha) [inline azure-pipelines.yml:230]: These tasks don't appear in the resulting changes. To confirm - these are being done by the release pipeline now?

Comment (erinha) [reply azure-pipelines.yml:230]: Confirmed offline that they are.

**Review by erinha — APPROVED** [review]
Thanks for making these improvements!

File: azure-pipelines.yml (hunk 1 above)
Comment (ramourt) [reply azure-pipelines.yml:230]: Correct

**Pull request merged**